src/lib/components/ProjectsPanel/components/Modal.module.css

```diff
@@ -0,0 +1,115 @@
+.modal {
+  transition: 0.2s all ease;
+  --footer-height: 80px;
+  font-family: "Iset Sans", sans-serif;
+  position: fixed;
+}
+
+.modal a {
+  color: inherit;
+  text-decoration: none;
+}
+
+.modal__overlay {
+  content: "";
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: rgba(0, 0, 0, 0.2);
+  z-index: -1;
+  opacity: 0;
+  visibility: hidden;
+  transition: 0.2s all ease;
+}
+
+.modal__content {
+  position: absolute;
+  bottom: 0;
+  padding: 16px 0 calc(var(--footer-height) + 8px);
+  visibility: hidden;
+  opacity: 0;
+  transition: 0.3s all ease;
+  transform: scale(0);
+  transform-origin: 20px calc(100% - 10px);
+  border-radius: 38px;
+}
+
+.modal__scroller {
+  overflow: auto;
+  max-height: 70vh;
+}
+
+.modal__divider {
+  height: 1px;
+  background-color: rgba(155, 170, 190, 0.3);
+  margin-left: 16px;
+  margin-right: 16px;
+  position: relative;
+  top: 8px;
+}
+
+.modal__subcontent {
+  position: absolute;
+  right: 16px;
+  bottom: -2px;
+  line-height: var(--footer-height);
+}
+
+.modal__subcontent a {
+  margin: 0 16px;
+  font-size: 32px;
```

Comment:
Let's try to make these `font-sizes` equivalent

- Common CSS variable with font value
- Make common parent with visible and hidden areas

![image](https://user-images.githubusercontent.com/22644149/215827790-07e6cfe0-9418-4eea-b691-8ee9d9285cff.png)

Reply:
will redesign with new icons